src/attach.js

```diff
@@ -19,11 +19,14 @@ function interceptor(fetch, ...args) {
   // Register fetch call
   promise = promise.then(args => {
     const request = new Request(...args);
+    const cloneRequest = request.clone();
     return fetch(request).then(response => {
```

Comment:
If Adding this  cloneRequest is not super cool let's do the following change at least
```suggestion
    return fetch(request.clone()).then(response => {
```